src/common/Input.ts

```diff
@@ -22,15 +22,81 @@ type KeyDescription = Required<
   Pick<KeyDefinition, 'keyCode' | 'key' | 'text' | 'code' | 'location'>
 >;
 
+/**
+ * Keyboard provides an api for managing a virtual keyboard.
+ * The high level api is {@link Keyboard.type | Keyboard.type},
```

Comment:
seems like it's getting tripped up on this link and not outputting it.

FYI if the thing after the | is the same as the thing before it, you don't need to provide it. `{@link Keyboard.type}` is the same.